build-system/babel-plugins/babel-plugin-mangle-object-values/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @fileoverview Mangles the values of an object expression.
+ * This is useful when defining large enums, as it allows us to use descriptive
+ * values during development, but use short mangled values during production.
+ */
+
+const {encode, indexCharset} = require('base62/lib/custom');
+
+const fnName = 'mangleObjectValues';
+
+module.exports = function (babel) {
+  const {types: t} = babel;
+
+  let charset;
+
+  /**
+   * @param {number} i
+   * @return {string}
+   */
+  function mangle(i) {
+    if (!charset) {
+      // Letters first since they allow unbracketed syntax when used as keys:
+      // (foo.a0 vs. foo['0a'])
```

Comment:
Nit: We're not worried about this, because they're string values not property keys. We can allow all printable ASCII range.

Reply:
The enum values can be used as keys for other objects, as it's the case with the current `StateProperty`.